createReleaseTarball.sh

```diff
@@ -89,13 +89,13 @@ sed -i '' -e s/"\"version\": \".*\""/"\"version\": \"${VERSION_WITHOUT_RC}\""/ p
 echo
 
 
-####################################
-#  REGENERATE npm-shrinkwrap.json  #
-####################################
-echo "[INFO] Removing lib/, node_modules/, and npm-shrinkwrap..."
-rm -rf lib/ node_modules/ npm-shrinkwrap.json
+############################
+#  REINSTALL DEPENDENCIES  #
+############################
+echo "[INFO] Removing lib/, and node_modules/..."
+rm -rf lib/ node_modules/
```

Comment:
Does it make sense to, since you're trying to clean install here, just do:

```bash
git clean -xdf
```

And just prune everything that doesn't exist by default in the git tree?

Reply:
This is a good suggestion. But I think we need to add a few more entries to  `.gitignore` before we do this. Will do as a separate PR.